### PR TITLE
Add composable

### PR DESCRIPTION
**Pull request by hoangdv2429**
This PR will add composable to app list

### CODE REVIEW

**Comment by hoangdv2429** [comment]
@delivan Can you review this PR ? Thank you

File: apps/composable/app.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "appCategory": "DeFi",
+    "appName": "Composable", 
+    "appSummary": "Composing DeFi for Mass Adoption - Any money, any chain, anywhere",
+    "appWebsiteUrl": "https://app.trustless.zone/",
```
Comment (josietyleung) [inline apps/composable/app.json:5]: https://www.composable.finance/`
can we use this please thanks!

Comment (hoangdv2429) [reply apps/composable/app.json:5]: @josietyleung hmm I'm debating between https://www.composable.finance/apps and the one in the script. fyi appWebsiteUrl is the link that lead to composable app. What do you think ?

Comment (delivan) [reply apps/composable/app.json:5]: IMO, Composable is more like a tool or infrastructure and https://app.trustless.zone is a DeFi app made of using Composable. So, `appWebsiteUrl` should be https://www.composable.finance` and `appCategory` should be `Tool`. How about this?

Comment (hoangdv2429) [reply apps/composable/app.json:5]: I agree @josietyleung wdyt ?

Comment (josietyleung) [reply apps/composable/app.json:5]: yes sounds good thanks!

**Comment by hoangdv2429** [comment]
Hi @delivan I updated it. I think it's ready to merge now 😄 thank you

**Pull request merged**